src/main.cpp

```diff
@@ -18,6 +19,7 @@
 #include <stdexcept>
 
 #include <Windows.h>
+#include <regex>
```

Comment:
Another small style thing. I like to keep standard library headers grouped separately from others.
```suggestion
#include <regex>
#include <stdexcept>

#include <Windows.h>
```